src/memory_manager.rs

```diff
@@ -350,6 +358,60 @@ pub fn get_allocator_mapping<VM: VMBinding>(
     mmtk.plan.get_allocator_mapping()[semantics]
 }
 
+/// Return an AllocatorInfo for the given allocator selector. This method is provided
+/// so that VM compilers may generate allocator fast-path and load fields for the fast-path.
+///
+/// Arguments:
+/// * `selector`: The allocator selector to query.
+pub fn get_allocator_info<VM: VMBinding>(selector: AllocatorSelector) -> AllocatorInfo {
```

Comment:
Better move the actual logic to `AllocatorInfo`. We try to keep methods in `memory_manager` simple -- mostly just forwarding calls. We have an intention to remove the module completely (https://github.com/mmtk/mmtk-core/issues/658).

Reply:
Should I implement `From<AllocatorSelector>`?

Reply:
Either looks good to me.